rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7"
```

Comment:
Cool. What should the package hierarchy be in the java code then?

Matching framework -> `com.jakewharton.rxbinding.support.v7.widget`

Matching library manifest -> `com.jakewharton.rxbinding.support.v7.recyclerview`

For added fun -> `com.jakewharton.rxbinding.support.v7.recyclerview.widget`